content/en/docs/performance_metrics/_index.md

```diff
@@ -23,16 +23,16 @@ The performance of a flow depends significantly on what the flow is doing. Conne
 
 |     |     |
 | --- | --- |
-| Date of test | November 2020 ([Agra](/docs/release_notes/agra)) |
+| Date of test | March 2022 ([Johannesburg](/docs/release_notes/johannesburg)) |
 | Number of requests | 100,000 |
 | Concurrency | 10 |
-| Total time | 115.498 seconds |
-| Bytes transferred | 61,000,001 |
-| Time per request (mean) | 11.550 ms |
-| Time per request (mean, across all requests) | 1.155 ms |
-| Memory idle | 12 MB |
-| Memory max | 12 MB |
-| **Requests per second** | **865.81** |
+| Total time | 75.487 seconds |
+| Bytes transferred | 54,200,001 |
```

Comment:
Up above it's talking about GreetFlow as a baseline which doesn't exist anymore so it's not clear what implementation is being used. There's also a very different quantity of data so i guess it's a different API?

Reply:
It was the Greet API.  It does exist if you know where to find it.  We may need to "generically" describe the API, e.g. "simple GET API with one parameter..."